app/core/icons/reload.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 22">
+  <path fill-rule="evenodd" d="M10.6515783,10.2903226 C10.1691483,10.2903226 9.79392496,9.91397849 9.79392496,9.40860215 C9.79392496,9.1827957 9.87969029,8.97849462 10.0405003,8.80645161 L12.9886837,5.87096774 C12.3990471,5.75268817 11.7343657,5.69892473 10.9946397,5.69892473 C6.97438952,5.69892473 3.76891007,8.91397849 3.76891007,12.9569892 C3.76891007,16.9892473 6.98511018,20.2258065 11.0053603,20.2258065 C15.0148898,20.2258065 18.2310899,16.9892473 18.2310899,12.9569892 C18.2310899,12.4301075 18.5741513,12.0430108 19.0887433,12.0430108 C19.6247767,12.0430108 20,12.4301075 20,12.9569892 C20,17.9677419 15.9904705,22 11.0053603,22 C6.00952948,22 2,17.9677419 2,12.9569892 C2,7.94623656 5.99880881,3.94623656 10.9946397,3.94623656 C11.5413937,3.94623656 12.0988684,3.98924731 12.6456224,4.08602151 L10.0297796,1.52688172 C9.87969029,1.34408602 9.79392496,1.12903226 9.79392496,0.903225806 C9.79392496,0.397849462 10.1691483,0 10.6515783,0 C10.919595,0 11.1232877,0.0967741935 11.2840977,0.268817204 L15.4544372,4.51612903 C15.6366885,4.69892473 15.7331745,4.93548387 15.7331745,5.17204301 C15.7331745,5.41935484 15.6474092,5.6344086 15.4544372,5.82795699 L11.2840977,10.0430108 C11.1232877,10.2043011 10.919595,10.2903226 10.6515783,10.2903226 Z"/>
+</svg>
```

Comment:
<img width="166" alt="image" src="https://user-images.githubusercontent.com/3460638/109732478-18f30500-7b72-11eb-8c26-cd09695225a9.png">

Reply:
An icon to refresh the summary data.

Reply:
This is for manually refreshing the summary dashboard. There is no "submit search" button to accomplish this, so I figured a refresh button like this would work for now.